src/kbucket/entry.rs

```diff
@@ -130,6 +130,7 @@ where
 
     /// Sets the status of the entry.
     pub fn update(self, state: ConnectionState, direction: Option<ConnectionDirection>) -> Self {
+        // TODO: action result ignored
```

Comment:
and here

Reply:
leaving this up since the return type has a `must_use` and I really don't know how this should be treated